interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -161,13 +175,18 @@
 }
 
 // Get total number of rows in the table.
-//
-$row = sqlQuery("SELECT COUNT(id) AS count FROM patient_data");
+// Include the custom filter clause and bound values, if any
+$row = sqlQuery("SELECT COUNT(id) AS count FROM patient_data WHERE $customWhere", $boundFilter->getBoundValues());
 $iTotal = $row['count'];
 
 // Get total number of rows in the table after filtering.
 //
-$row = sqlQuery("SELECT COUNT(id) AS count FROM patient_data $where", $srch_bind);
+if (empty($where)) {
+    $where = $customWhere;
+} else {
+    $where = "$customWhere AND $where";
```

Comment:
I think this needs to be flipflopped to ensure the ordering of the binds is correct (since the $where stuff was binded in the array first)

Reply:
oh wait, scrap my comment above, you did these first in the array merge